src/strategies/strategy-base.sol

```diff
@@ -358,20 +362,21 @@ abstract contract StrategyBase {
 
     function _distributePerformanceFeesAndDeposit() internal {
         uint256 _want = IERC20(want).balanceOf(address(this));
+        uint256 _want_treasury = _want.mul(performanceTreasuryFee).div(performanceTreasuryMax);
```

Comment:
Should we `require(_want > 0)` above as a guard clause instead to save a bit more by avoiding the mul/div calls?

Reply:
If performance and dev fees were set to 0 wouldn't that make the entire harvest tx revert?  Do you mean to wrap the whole thing in `if (_want > 0)` ?

Reply:
Sorry, yes, didn't read it properly